packages/calcite-components/src/components/action-bar/action-bar.e2e.ts

```diff
@@ -368,6 +368,43 @@ describe("calcite-action-bar", () => {
       expect(await page.findAll(slottedActionsSelector)).toHaveLength(7);
     });
 
+    it("should slot 'menu-actions' on sublist changes after being enabled", async () => {
+      const page = await newE2EPage({
```

Comment:
In an upcoming PR, all tests will be creating the E2E page and then setting the content. Would you mind updating this test to follow that pattern?